compiler/tests-ocaml/lib-effects/double-translation/dune

```diff
@@ -1,12 +1,12 @@
 (env
- (using-effects
+ (with-effects
   (flags
    (:standard -w -38))
   (js_of_ocaml
    (flags
-    (:standard --effects=double-translation))
+    ((:standard \ --effects=cps) --effects=double-translation))
    (build_runtime_flags
-    (:standard --effects=double-translation))
+    ((:standard \ --effects=cps) --effects=double-translation))
    ;; separate compilation doesn't work when using
```

Comment:
We should update the comment to say that dune does not know about `--effects=double-translation`.

Reply:
The comment is stale. separate compilation now works with `--enable use-js-srtring` or `--enable effects`.

Reply:
> The comment is stale. separate compilation now works with --enable use-js-srtring or --enable effects.

> ```
> ;; separate compilation doesn't yet work when using '--effect=double-translation' since dune doesn't know it should compile a different version of the dependencies
> ``
> ```

Done.